gp-multi-page-navigation/gpmpn-dropdown-navigation.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Gravity Perks // Multi-page Navigation // Enable "Back to Last Page" Button for Progress Bar Navigation
+ * https://gravitywiz.com/documentation/gravity-forms-multi-page-navigation/
+ *
+ * Instruction Video: https://www.loom.com/share/38fcc941036842d6a81a599fd8de39d7
+ *
+ * This snippet is designed to be used with the Gravity Forms Code Chest plugin.
+ * https://gravitywiz.com/gravity-forms-code-chest/
+ */
+function updateNavigationState(  ) {
+	// hide if visible
+	$( '#gf_page_steps_GFFORMID' ).attr( 'hidden', 'true' );
+
+	var $select = $( '#page_steps_dropdown' );
+
+	// Check if the select element already exists
+	if ( $select.length === 0 ) {
+		// Create a new select element if it does not exist
+		$select = $( '<select id="page_steps_dropdown"></select>' );
+		$( '#gf_page_steps_GFFORMID' ).after( $select );
+
+		$select.on( 'change', function() {
+			var selectedValue = $( this ).val(  );
+			if ( selectedValue ) {
+				var $matchingLink = $( '#gf_page_steps_GFFORMID .gf_step a[href="' + selectedValue + '"]' );
+				if ( $matchingLink.length > 0 ) {
+					$matchingLink[0].click();
+				}
+			}
+		} );
+	} else {
+		$select.empty();
+	}
+
+	$( '#gf_page_steps_GFFORMID .gf_step' ).each( function(  ) {
+		var $div = $( this );
+		var stepNumber = $div.find('.gf_step_number').text();
+		var $a = $div.find('a');
+		var $option = $('<option></option>').text( 'Step ' + stepNumber );
+
+		if ( $a.length > 0 ) {
+			// found a tag, add the link to the dropdown option
+			var href = $a.attr( 'href' );
+			$option.attr( 'value', href );
+		} else if ( $div.hasClass( 'gpmpn-step-current' ) ) {
+			// current step should remain selected, but no link needed
+			$option.attr( 'selected', 'selected' );
+		} else {
+			// page not yet traversed, should be disabled
+			$option.attr( 'disabled', 'disabled' );
+		}
+
+		$select.append( $option );
+	} );
+}
+
+// Initialize the navigation state on page load
+setTimeout( updateNavigationState, 200 );
```

Comment:
I didn't need the 200 in my testing. What issue did you see?

Reply:
Sorry, @saifsultanc, I meant keeping the setTimeout but removing the 200. 

```
setTimeout( updateNavigationState );
```

Reply:
@spivurno Okay yeah. We can do that. That did work. But, I was being extra cautious with the page render. Will push the update to the PR.